crypto/ec/ec_mult.c

```diff
@@ -222,11 +222,11 @@ int ec_wNAF_mul(const EC_GROUP *group, EC_POINT *r, const BIGNUM *scalar,
 
     totalnum = num + numblocks;
 
-    wsize = OPENSSL_malloc(totalnum * sizeof wsize[0]);
-    wNAF_len = OPENSSL_malloc(totalnum * sizeof wNAF_len[0]);
-    wNAF = OPENSSL_malloc((totalnum + 1) * sizeof wNAF[0]); /* includes space
+    wsize = OPENSSL_malloc(totalnum * sizeof(wsize[0]));
+    wNAF_len = OPENSSL_malloc(totalnum * sizeof(wNAF_len[0]));
+    wNAF = OPENSSL_malloc((totalnum + 1) * sizeof(wNAF[0])); /* includes space
                                                              * for pivot */
```

Comment:
Jagged comment.

Reply:
I fixed this the same way I did in master (put it on previous line) and pushed and updated commit.  I tend to do "feature-creep" PR's and @levitte tends to call me out on them, so this is just fixing sizeof syntax.

Reply:
You killing me, man. You know I don't have no magic coin, so with no separate fixup commit I have to start over...

Reply:
I'll try to do better in the future.

Reply:
I actually didn't catch that in master, sorry.

I agree that the indentation should be corrected...  and maybe you and I should talk, 'cause it seems we have different ideas of what constitutes "feature creep"

Reply:
I think I generally agree with levitte about feature creep, with the justification that for this sort of sweeping change, it may actually be easier to review by having an independent implementation of the change and comparing the implementations -- the human eye tires too easily looking at a long stream of "the same" change.

Reply:
The indentation in master is fixed, the same way fixed here, by putting the entire comment on the line above.

Reply:
As for the "feature creep" comment, I was trying more for lighthearted self-deprecation.

Reply:
Ah, ok...  sorry, tired day today, didn't catch the lightheartedness (obviously)